src/applications/vaos/tests/appointment-list/components/FutureAppointmentsList.video.unit.spec.jsx

```diff
@@ -866,10 +865,7 @@ describe('VAOS <FutureAppointmentsList> calendar ics file format', () => {
       (_, node) => node.textContent === 'VA Video Connect at a VA location',
     );
 
-    // wait for facilities fetch
-    await waitFor(() => {
-      expect(screen.findByText('Cheyenne VA Medical Center')).to.be.ok;
-    });
+    expect(await screen.findByText('Cheyenne VA Medical Center')).to.be.ok;
```

Comment:
Ah, thanks for removing the unneeded `waitFor` I added.

Reply:
It's interesting that it sort of worked, I guess because it waited a tick or something. The ultimate issue was there needed to be an `await` in front of `screen.findByText`